Sources/tuist/main.swift

```diff
@@ -5,6 +5,9 @@ import TuistLoader
 import TuistSupport
 
 if CommandLine.arguments.contains("--verbose") { try? ProcessEnv.setVar(Constants.EnvironmentVariables.verbose, value: "true") }
+if CommandLine.arguments.contains("--generate-completion-script") {
+    try? ProcessEnv.unsetVar(Constants.EnvironmentVariables.silent)
```

Comment:
If we add any logging in this module on the "path" to generating the completion script, it will break again.

Can we do a more precise operation - for example, what happens if you change the `logLevel` of the `Logger` defined [here](https://github.com/tuist/tuist/blob/main/Sources/TuistKit/Log/Logger.swift)? We should explore if there's a way how to change the `logLevel` _after_ bootstrapping the logging.

Reply:
So you're proposing to keep the `logLevel` at  `error` as long as possible and lower it to `info` _just_ before the completion script is outputted at `info` level?

Reply:
Yes, exactly. If a contributor adds some unexpected log, we want to suppress it by default for the script generation.